frontend/src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -228,7 +228,9 @@ export const ProjectAccessTable: VFC = () => {
                             permission={UPDATE_PROJECT}
                             projectId={projectId}
                             to={`edit/${
-                                row.type === ENTITY_TYPE.USER ? 'user' : 'group'
+                                row.type === ENTITY_TYPE.GROUP
+                                    ? 'group'
+                                    : 'user'
```

Comment:
Should this `'user'` be a `'service account'` or is it ok to aggregate both under `user` in this case?

Reply:
Nice catch! In this case it's important that both types are handled as users, as both types should be handled by `frontend/src/component/project/ProjectAccess/ProjectAccessEditUser/ProjectAccessEditUser.tsx`